models/staging/stg_fivetran_log__log.sql

```diff
@@ -1,3 +1,11 @@
+{{ config(
+    partition_by={
+        'field': 'created_at',
+        'data_type': 'timestamp',
+        'granularity': 'day'
+    }
+) }}
+
```

Comment:
We should remove this as it was just for testing purposes for [Dylan](https://github.com/fivetran/dbt_fivetran_log/pull/29#discussion_r778714632).

Instead we should maybe add documentation to the CHANGELOG to highlight how users can add a partition on an existing table.

Reply:
removed the code! would pointing folks to the fivetran [docs](https://fivetran.com/docs/destinations/bigquery/partition-table) on how to convert to partitioned tables fit what you're thinking of?

Reply:
Yeah that would be perfect!